tasks/eva_3754/categorise_duplicate_candidates.py

```diff
@@ -0,0 +1,88 @@
+import datetime
+import itertools
+import json
+import sys
+from argparse import ArgumentParser
+from collections import Counter
+
+
+def convert_date(datetime_dict):
+    date_tmp = {}
+    date_tmp.update(datetime_dict.get('date'))
+    date_tmp.update(datetime_dict.get('time'))
+    date_tmp['microsecond'] = int(date_tmp.pop('nano')/1000)
+    return datetime.datetime(**date_tmp)
+
+def min_time_gap_between_create_date(cve_list):
+    date_list = [convert_date(cve.get('createdDate')) for cve in cve_list]
+    delta_dates = [abs(d1-d2) for d1,d2 in  itertools.combinations(date_list, 2)]
+    min_delta_date = min(delta_dates)
+    return min_delta_date
+
+def more_than_one_assembly(cve_list):
+    assembly_count = Counter([cve.get('assemblyAccession') for cve in cve_list])
+
+    return ','.join(sorted([assembly for assembly in assembly_count if assembly_count[assembly] > 1]))
```

Comment:
This is outputting something different from "Are all of these CVEs in one assembly or not", and it's a bit harder for me to interpret - e.g. cases where there are multiple assemblies, some of which might contain multiple CVEs and some might not. It doesn't matter in the end if the root causes/explanations are similar, but just something to point out...

Reply:
That's true the name of the function is wrong and should say something like. assemblies with more than one CVE.